Sources/Valet/Internal/Service.swift

```diff
@@ -42,9 +42,19 @@ internal enum Service: CustomStringConvertible, Equatable {
     internal func generateBaseQuery() -> [String : AnyHashable] {
         var baseQuery: [String : AnyHashable] = [
             kSecClass as String : kSecClassGenericPassword as String,
-            kSecAttrService as String : secService
+            kSecAttrService as String : secService,
         ]
-        
+
+        #if os(macOS)
+        if #available(macOS 10.15, *) {
```

Comment:
I _only_ add `kSecUseDataProtectionKeychain` to our base query on macOS, since macOS is the only OS that _needs_ this value.

Adding this value to the base query is a breaking change, since anything written to the keychain without `kSecUseDataProtectionKeychain` set will not be read when `kSecUseDataProtectionKeychain` is in the base query. I wrote `migrateObjectsFromPreCatalina` to solve this issue on macOS, but decided to allow non-Mac customers to continue using the keychain without migrating objects to include `kSecUseDataProtectionKeychain`.

Reply:
After sleeping on it: I realize we can use `kSecUseDataProtectionKeychain` on all platforms. I had [written a test to prove this](https://github.com/square/Valet/pull/193/files#diff-4919d5fe96673b6a2432ee8dc3bbb663R307).

Reply:
From [Apple's documentation](https://developer.apple.com/documentation/security/ksecusedataprotectionkeychain):
> It’s highly recommended that you set the value of this key to true for all keychain operations. This key helps to improve the portability of your code across platforms. Use it unless you specifically need access to items previously stored in a legacy keychain in macOS.

Fixed in https://github.com/square/Valet/pull/193/commits/0c9799c491d7b77507be2a0717018ff000b0c0d7